rest/account_apikey_test.go

```diff
@@ -68,4 +88,43 @@ func TestCreateDDIAPIKey(t *testing.T) {
 
 	_, err := c.APIKeys.Create(k)
 	require.NoError(t, err)
+	// Create a key with auth tags
+	k = &account.APIKey{
```

Comment:
Might also be valuable to include a key in where permissions are empty like:
```go
	account.APIKey{
		ID:   "foo",
		Key:  "foo-1",
		Name: "foo-1",
		Permissions: account.PermissionsMap{
			DHCP: &account.PermissionsDHCP{},
			IPAM: &account.PermissionsIPAM{},
		},
	}
```

Reply:
good call!